lib/lib.go

```diff
@@ -791,8 +791,10 @@ func (inst *Instance) Connect(ctx context.Context) (err error) {
 	return nil
 }
 
-// Config provides methods for manipulating Qri configuration
-func (inst *Instance) Config() *config.Config {
+// GetConfig provides methods for manipulating Qri configuration
+// TODO (ramfox): DEPRECATED - we are moving to a paradigm where the only methods
+// on inst are the accessors
```

Comment:
Use the [golang deprecation format](https://github.com/golang/go/wiki/Deprecated). In addition, depreciation notices should always tell a user what to use instead.  Should look like this:

```go
// GetConfig provides methods for manipulating Qri configuration
// 
// Deprecated: this method will be removed in a future release.
// Use inst.Config().GetConfig instead
func (inst *Instance) GetConfig() *config.Config {
```

The fact that `inst.Config().GetConfig` isn't the exact same means we should consider moving `GetConfig` to config methods (`inst.Config().Get`), but let's save that for a follow-up PR